pkg/provisioner/reconciler/claim-reconciler/reconiler.go

```diff
@@ -144,6 +143,10 @@ func (r *ObjectBucketClaimReconciler) handleProvisionClaim(key client.ObjectKey,
 		return err
 	}
 
+	// use storage class to determine if this bkt was dynamically provision
+	// (greenfield) or not (brownfield)
```

Comment:
nits 
- `bkt was dynamically provision` should read `bucket should dynamically provisioned`
- "greenfield" and "brownfield" aren't very descriptive of the cases we're dealing with and could be hard to interpret by other devs (assuming anyone besides us ever reads this code :P )